mujoco_warp/_src/solver.py

```diff
@@ -376,11 +426,18 @@ def _init_p0(p0: wp.array(dtype=wp.vec3), d: types.Data):
       if d.efc.done[worldid]:
         return
 
-    # TODO(team): active and conditionally active constraints:
-    if d.efc.Jaref[efcid] >= 0.0:
+    Jaref = d.efc.Jaref[efcid]
+    if (Jaref >= 0.0) and (efcid >= d.ne[0] + d.nf[0]):
       return
 
     quad = d.efc.quad[efcid]
+    jv = d.efc.jv[efcid]
+
+    if wp.static(m.dof_frictionloss_adr.size > 0 and (not DSBL_FLOSS)):
+      floss = d.efc.frictionloss[efcid]
+      efc_D = d.efc.D[efcid]
+      quad = _quad_floss(quad, floss, efc_D, Jaref, jv)
```

Comment:
it seems like you're calculating this _quad_floss a lot - is it possible to just store this once in `efc.quad` when calling _init_quad?